src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\node\Plugin\EntityReferenceSelection\OgSelection.
+ */
+
+namespace Drupal\og\Plugin\EntityReferenceSelection;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\Plugin\EntityReferenceSelection\DefaultSelection;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\og\Og;
+
+/**
+ * Provide default OG selection handler.
+ *
+ * @EntityReferenceSelection(
+ *   id = "default:og",
+ *   label = @Translation("OG selection"),
+ *   entity_types = {"node"},
+ *   group = "default",
+ *   weight = 1
+ * )
+ */
+class OgSelection extends DefaultSelection {
+
+  private $targetType;
+
+  /**
+   * @var AccountInterface
+   *
+   * The current user object.
+   */
+  protected $account;
+
+  /**
+   * Get the current account.
+   *
+   * @return AccountInterface
+   */
+  public function getAccount() {
```

Comment:
We have a property called `currentUser` but with no setter or getter(the current account set with a dependancy injection)